src/pages/pricing.tsx

```diff
@@ -1,263 +1,288 @@
+                cta1={{
+                    text: 'Deploy',
+                    link: '/get-started/self-hosted',
+                    ctaStyle: 'primaryButton',
+                }}
+                cta2={{
+                    text: 'Start free Cloud trial',
+                    link: '/demo',
+                    ctaStyle: 'outlineButton',
+                }}
+            />
 
-                <ContentSection>
-                    <Blockquote
-                        border={false}
-                        quote="Sourcegraph pays for itself many times over—it's a game changer."
-                        author="François Jehl, Senior Engineering Manager, Criteo"
-                        logo={{
-                            src: '/external-logos/criteo-logo.svg',
-                            alt: 'Criteo logo: Read the case study',
-                            href: '/case-studies/criteo-tackles-big-code',
-                        }}
-                    />
-                </ContentSection>
-            </div>
+            <ContentSection className="tw-grid tw-grid-cols-5">
+                <h2 className="tw-col-span-full md:tw-col-span-2 tw-mb-md tw-max-w-md">Frequently asked questions</h2>
 
-            <div className="tw-border-t tw-border-t-gray-200">
-                <CtaSection
-                    title="Try Sourcegraph for free today"
-                    description="You'll be searching your own code in 10 minutes. You can run it self-hosted (all of your code stays local and secure)."
-                    cta1={{
-                        text: 'Try Sourcegraph now',
-                        link: '/get-started/self-hosted',
-                        ctaStyle: 'primaryButton',
-                    }}
-                    cta2={{
-                        text: 'Schedule a demo',
-                        link: '/demo',
-                        ctaStyle: 'outlineButton',
-                    }}
-                />
-            </div>
-        </div>
-    </Layout>
-)
+                <div className="tw-col-span-full md:tw-col-span-3">
+                    <Accordion>
+                        {faqData.map((item, index) => (
+                            <Accordion.Item
+                                key={item.q}
+                                eventKey={item.q}
+                                className="tw-border-t-1 tw-border-gray-200 tw-max-w-2xl tw-mb-sm"
+                            >
+                                <Accordion.Header onClick={() => setActiveKey(activeKey !== index ? index : null)}>
+                                    <div className="tw-flex tw-justify-between tw-items-center">
+                                        <h4 className="tw-text-start tw-mt-sm">{item.q}</h4>
+                                        <span className="tw-text-gray-400 tw-font-normal tw-mt-xs">
+                                            {activeKey === index ? <MinusIcon /> : <PlusIcon />}
```

Comment:
Bootstrap 4 doesn't have a toggle icon for Accordion. Once we convert this to Flowbite this logic will get flatter.

Reply:
Sounds good!